### ISSUE TITLE
Funding problem is false claims

### ISSUE THREAD

**Issue by philmmanjaro** [EmeraldSnorlax/manjarno#32]
On the page you have the following statement:

>Funding
>
>Manjaro has had a controversy with their treasurer. Phillip Muller (Manjaro team lead) had purchased a laptop for €2000, and the treasurer asked to clarify his purchase. [This ultimately led to the treasurer being removed.](https://redd.it/hxp3zi) Isn’t the whole point of a treasurer to ensure fair and efficient use of donation funds?

Maybe recheck the claims that I purchased a 2000 € laptop and see who actually did: https://opencollective.com/manjaro/expenses/22477 Also it was not approved by me in the end, rather by another member of the financial committee from the Manjaro Linux project.

The original post by Jonathon can be found here:  https://redd.it/hwo33h

Also, Jonathon was just the initial admin of the groups he created on the fiscal hosts. The actual treasurer is on the fiscal host side to finally approve any spending based on the policies a project has.

```
# Expense Policy (v. 0.10) 17.Aug.2020
1. The goal of the Manjaro project is to provide a Linux distribution free of charge.
2. Expenses will be covered for costs which support and progress the Manjaro project, for example and non-exhaustively:

* Travel and subsistence costs for conferences and related activities;
* Hardware purchases and upgrades which support the development of Manjaro;
* Software purchases, upgrades, and licenses which support the development of Manjaro;
* Hosting and SaaS/PaaS etc. charges related to the project.
* Setting bounties on bugs or features for Manjaro software 

3. Expenses should be discussed with members of the Manjaro Team to determine that expenses are for approriate purpose and that enough funds remain to sustain the distribution infrastructure for at least five following years. 
4. An internal financial committee of three Manjaro Team members who will rotate on yearly basis is responsible for facilitating the discussion, informing about the vote, assuring a valid vote process, fixating the validity of the vote, executing the vote decisions as fund admins on fund management platforms.
5. Depending on the amount, expenses must be approved by a vote of a certain number of members of the Manjaro Team.
For expenses < 400€ no vote is needed, fund admins will question in suspicious cases,
for expenses >= 400€ and < 1000€ three Manjaro Team members must vote,
for expenses >= 1000€ five Manjaro Team members must vote, 
in order to make the vote valid. When the number of team members changes, these numbers are adjusted accordingly.
6. When a team member notices that the minimum amount of voters have voted and the vote was in favour of the expense, the team member should make a screenshot, add a time stamp, notify the financial committee and after 12 hours the vote is declared valid by at least one of the members of the financial committee, the expense filed and approved on a fund management platform.
7. A valid vote with a simple majority in favour of the expense results in an approval of the expense.
8. The invoice for the expense should be filed at a fund management platform either by the applicant or a Manjaro Team member on behalf of the applicant, but not by any of the current internal financial committee members.
9. Normally a vote should be organized prior to any purchases being made, before being submitted to the fund management platform. If an expense is submitted before vote has been organized, internal financial committee is responsible for organizing the vote prior to approving the expense.
10. The expenses are transparent by design on the chosen fund management platform.
11. This policy can be revised and changed at any time by a simple majority of the Manjaro team. The vote for a change of the policy should be left open for 3 days.
```

Since expenses are transparent on [OpenCollective](https://opencollective.com/manjaro/expenses?orderBy=CREATED_AT%2CASC) and [LFX](https://crowdfunding.lfx.linuxfoundation.org/projects/manjaro/financial) each process can be verified ...

**Comment by EmeraldSnorlax** [comment]
wow, thats quite the colossal fuck up on behalf of me. thank you so much for reaching out to me.
im really sorry for this and will ammend the section immediately (but will leave a link to this thread for transparency's sake)

**Comment by dataprolet** [comment]
It is more than weird that you bring this up now after Jonathon has passed and stir up confusion about this issue once more.

**Comment by Silejonu** [comment]
As @dataprolet said, the timing at which you're disputing it can only seem fishy at best. This claim has been on the website [for a long time](https://web.archive.org/web/20210929033039/https://manjarno.snorlax.sh/) *(2021-09-29 as far as the Internet Archive goes)*, and it is unlikely you weren't aware of it earlier.

To add some context:

- Jonathon [passed away recently](https://lists.archlinux.org/archives/list/aur-general@lists.archlinux.org/thread/UA6NYEGLXKTDW6PE22E3EJTGB27O46XR/)
- In [this thread](https://web.archive.org/web/20200730174817/https://forum.manjaro.org/t/change-of-treasurer-for-manjaro-community-funds/154888), he was explaining he was no longer the Manjaro "**treasurer**". Yet when [you publicly answered](https://web.archive.org/web/20200807041420/https://forum.manjaro.org/t/change-in-manjaro-team-composition/155231), at no point did you refute his claim of him being the treasurer. However, you described him as "the administration of donations". Note that all previous Manajaro expenses [have been approved by Jonathon](https://opencollective.com/manjaro/expenses?period=all%E2%86%922020-07-29T21%3A59%3A59.999Z) (for those in which the person approving is listed, at least).
- Jonathon seemed to imply that [buying the laptop with community funds to accommodate for a deal between the Manjaro company and the manufacturer of the laptop](https://www.reddit.com/r/ManjaroLinux/comments/hwo33h/comment/fz32iby/?utm_source=share&utm_medium=web2x&context=3) was what pushed him to refuse the transaction in the first place.
- Regarding "for expenses >= 1000€ five Manjaro Team members must vote", is there any record of the vote having taken place?

I'm inclined to believe what you're saying, but what's with this terrible timing? Why are you disputing it at least a year and a half after this has been published on this page? Is there more context that we're not aware of?

**Comment by philmmanjaro** [comment]
Yes it was voted in our internal part of the forum. Those things won't get voted in the public as it is internal matters of the project members. Why should we complicate things and include the user community into those decisions?

Here you go:

![image](https://user-images.githubusercontent.com/5050022/229266158-1074546a-b023-401f-b674-039696fd047f.png)

The main issue is how donations are used and how the process works. We are in a world where everyone has to have money and then get reimbursed to get it back. Also it took a lot of discussions back then to even ask for new hardware within the team. I was simply sick of that fact.  In the end it was actually an enabler to spend the money on new gear as shown [here](https://opencollective.com/manjaro/expenses?searchTerm=Laptop).

Having deals with hardware vendors like Tuxedo and others doesn't mean that we have to use them in that particular case. If you want to support Nvidia Prime you may need a Gaming Laptop for that. Some of our vendors lent Notebooks, others reduce the prices. You have to decide case to case.

Jonathon initially set up the fiscal hosts back then. I was busy to start the company. All the old funds are still in my bank accounts due to tax reasons. Also back then the team voted and I released the money.

Anyhow it really doesn't matter how a project spends money when it doesn't do it at all.

Why to dispute now? No particular reason. I was simply in the mood to deal with some old social media rumors  popping up and up and everybody is adding his take on it.
